WORKSPACE

```diff
@@ -38,21 +38,21 @@ bind(
 # 2. Update .bazelversion, envoy.bazelrc and .bazelrc if needed.
 #
 # istio/envoy commit date: Jun 02 2020
```

Comment:
nit: Is it require to update this date also if the SHA is updated?

Ref: Last PR https://github.com/istio/proxy/pull/2867/files#diff-09498dbadf45966909850dc8a47ebb13R40

Reply:
It was an oversight, but given that it's just a comment and it already got merged it's fine. Making sure the correct sha is updated is more important. Didn't notice that.